### DISCUSSION TITLE
API OLT ZTE

### DISCUSSION THREAD

**Discussion by syfrrn** [community/community#40695] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi,

Has anyone here used zte-libraries-php by xeleniumz ?

link : https://github.com/xeleniumz/zte-libraries-php

how to integrate this library into CodeIgniter v4 ?
please help me

**Comment by jfullstackdev** [comment]
seems the project update was last 2017, although the developer is still active in different projects, maybe you can ask him the same question in one of his active repos
